src/format/command.rs

```diff
@@ -10,8 +12,44 @@ pub fn run(stdin: bool, check: bool, files: Vec<String>) -> Result<(), Error> {
     }
 }
 
-pub fn format_files(_files: Vec<String>, _check: bool) -> Result<(), Error> {
-    todo!()
+pub fn format_files(files: Vec<String>, _check: bool) -> Result<(), Error> {
+    for file_path in files {
+        let path = PathBuf::from_str(&file_path).map_err(|e| Error::FileIO {
+            action: FileIOAction::Open,
+            kind: FileKind::File,
+            path: PathBuf::new(),
+            err: Some(format!("Invalid path: {}\n{}", file_path, e.to_string())),
```

Comment:
Can we use the error `e` here rather than using our own error? The printed output says this this field is from the underlying library and I don't want to lose any information here.

Reply:
I _think_ I addressed this.